src/OrbitGl/Track.cpp

```diff
@@ -85,8 +85,7 @@ void Track::DrawTriangleFan(Batcher* batcher, const std::vector<Vec2>& points, c
 void Track::Draw(GlCanvas* canvas, PickingMode picking_mode, float z_offset) {
   CaptureViewElement::Draw(canvas, picking_mode, z_offset);
 
-  Batcher* batcher = canvas->GetBatcher();
-
+  Batcher* ui_batcher = canvas->GetBatcher();
```

Comment:
Complete nitpick, feel free to ignore: The naming "ui_batcher" to me infers more knowledge than the track should really need, I thought "batcher" was fine before. I mostly stumbled across this because the `DrawTriangleFan` lines got reformatted :D

Reply:
mmm, I will disregard this. We used to have some issues with using both batchers without understanding which we had to use. 

I will change this after removing canvas here.